oauth2client/contrib/django_util/views.py

```diff
@@ -22,20 +22,21 @@
 import hashlib
 import json
 import os
-import pickle
 
 from django import http
 from django import shortcuts
 from django.conf import settings
 from django.core import urlresolvers
 from django.shortcuts import redirect
+import jsonpickle
 from six.moves.urllib import parse
 
 from oauth2client import client
 from oauth2client.contrib import django_util
 from oauth2client.contrib.django_util import get_storage
 from oauth2client.contrib.django_util import signals
 
+
```

Comment:
I don't have a preference here but I think @nathanielmanistaatgoogle wants 1 line between imports and the constants? I dunno.